paymentsheet/src/main/java/com/stripe/android/paymentsheet/FlowControllerCompose.kt

```diff
@@ -84,6 +84,24 @@ fun rememberPaymentSheetFlowController(
     )
 }
 
+/**
+ * Creates a [PaymentSheet.FlowController] that is remembered across compositions.
+ *
+ * @param builder which contains required [PaymentOptionCallback] and [PaymentSheetResultCallback] as well as
+ * other optional callbacks.
+ */
+@Composable
+fun rememberPaymentSheetFlowController(
```

Comment:
I don't feel strongly here, but this feels a little less ergonomic than calling `build`. Since when I have a builder, I typically expect to call build to get a thing, rather than pass it to another method which actually does the "building".

cc @tillh-stripe

Reply:
Yeah I would prefer to have a composable build as well. It would also make it easier to have one code path for initializing callbacks

Reply:
Updated